styles/variables-base.less

```diff
@@ -306,7 +306,7 @@
 @agate-heading-font-ligatures: @agate-base-font-ligatures;
 @agate-heading-title-height: auto;
 @agate-heading-title-line-height: inherit;
-@agate-heading-title-padding: @agate-heading-padding 50px;
+@agate-heading-title-padding: 0 51px 21px 51px;
```

Comment:
I think we should add margin-padding only when showline.
and Please add screenshot test for showline.

Reply:
I don't see why I should delete the padding-bottom here, sorry if I don't understand correctly. It would look like this:
![heading-padding](https://user-images.githubusercontent.com/63335823/122551973-61a9cd80-d03e-11eb-9dcc-97bd64a5a8ff.PNG)

_It is also a bug that showline =false is not woking on the sampler._
Could you give more details? I see no problem with `showline` prop

Reply:
Added more ss tests

Reply:
It's my misunderstand.
I found that the center line and showline have nothing to do with it.